src/python_ml_project_template/datasets/flow_trajectory_dataset.py

```diff
@@ -0,0 +1,193 @@
+import copy
+from typing import Dict, List, Literal, Optional, Sequence, TypedDict, Union
+
+import numpy as np
+import numpy.typing as npt
+import rpad.partnet_mobility_utils.articulate as pma
+import rpad.partnet_mobility_utils.dataset as pmd
+from rpad.partnet_mobility_utils.data import PMObject
+
+
+class FlowTrajectoryData(TypedDict):
+    id: str
+    pos: npt.NDArray[np.float32]  # (N, 3): Point cloud observation.
+    trajectory: npt.NDArray[np.float32]  # (N, K, 3): Ground-truth flow.
+    mask: npt.NDArray[np.bool_]  #  (N,): Mask the point of interest.
+
+
+def compute_normalized_flow(
```

Comment:
Can you make a note here about what changes you made from the original function?